pkgs/development/python-modules/esper/default.nix

```
@@ -0,0 +1,20 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core
+}:
+buildPythonPackage rec {
+  pname = "esper";
+  version = "3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "benmoran56";
+    repo = "esper";
+    rev = "v${version}";
```

Avoid branch/tag confusion, especially useful for batch updates.

```suggestion
    rev = "refs/tags/v${version}";
```